.gitmodules

```diff
@@ -2,6 +2,7 @@
 path = awsm
 url = https://github.com/gwsystems/aWsm
 ignore = dirty
+branch = sledge-compat
```

Comment:
Oh another branch on awsm. Feels like we're going to need a doc for keeping track of all the branches haha

Reply:
Hopefully should be able to get merged. It's just a bit slower to merge stuff over there generally.